readme.html

```diff
@@ -17,5 +17,14 @@
           <li>A new entry with the Name you provided appears on the page. Expand it and click on 'Show Token' to get the token</li>
           <li>Copy the generated token and use it in 'Access Token' asset configuration parameter</li>
         </ol>
+      <p><h2>Event Dimensions</h2>
+        <ol>
+          When crafting your Event Dimensions keep in mind:
+          <li>Must be valid JSON</li>
+          <li>curly bracket characters that are not datapath references must be escaped with an additional curly bracket. E.G. { becomes {{</li>
```

Comment:
Hi @greatestusername-splunk ,
We have started reviewing this PR. We are trying to understand the need for escaping values with a curly brace. Could you please help us understand it with some example dimension values which will and will not require escaping?

Reply:
Hello Dvora!

Thanks for letting me know!

What is being entered into that box is a dictionary of dimensions. The reason it must be a dict is because we are constructing the `dimensions` dict that is used when creating an event via the API. 

Example CURL from the docs to show what that `dimensions` dict looks like:
```
curl -X POST "https://ingest.{REALM}.signalfx.com/v2/event" \
    -H "Content-Type: application/json" \
    -H "X-SF-Token: <value>" \
    -d '[
            {
                "category": "USER_DEFINED",
                "eventType": "test_event",
                "dimensions": {
                    "environment": "production",
                    "service": "API"
                },
                "properties": {
                    "sha1": "1234567890abc"
                },
                "timestamp": 1556793030000
            }
        ]'
```

the escaping is required so that the `{` and `}` characters of the dict are accurately represented.

Example working input (with escapes):
`{{"key1":"value1", "key2":"value2"}}`

Example non-working input (no escapes):
`{"key1":"value1", "key2":"value2"}`

Let me know if you have any other questions? Thanks!!

Reply:
further example with a datapath `{0}` in it

`{{"key1":"value1", "key2":"{0}"}}`
<img width="523" alt="image" src="https://user-images.githubusercontent.com/83726369/203378653-e026b126-4d15-49f8-96b4-5a28a1810e6d.png">

Reply:
Verified that the double curly brackets are only required when using data paths.

Reply:
Thanks for the explanation @greatestusername-splunk